pkg/cloudprovider/aws/fleet/launchtemplate.go

```diff
@@ -129,3 +140,28 @@ func (p *LaunchTemplateProvider) getSecurityGroupIds(ctx context.Context, cluste
 	}
 	return securityGroupIds, nil
 }
+
+func (p *LaunchTemplateProvider) getAMIID(context context.Context) (*string, error) {
+	version, err := p.kubeServerVersion()
+	if err != nil {
+		return nil, fmt.Errorf("kube version %w", err)
+	}
+	paramOutput, err := p.ssm.GetParameter(&ssm.GetParameterInput{
+		Name: aws.String(fmt.Sprintf("/aws/service/eks/optimized-ami/%s/amazon-linux-2/recommended", version)),
+	})
+	if err != nil {
+		return nil, fmt.Errorf("ssm get parameter %w", err)
+	}
+	output := struct {
```

Comment:
This is crazy! Does the Go CDK really not provide you these types?

Reply:
Couldn't find a type for the `Value` filed in the parameterOutput struct.